.github/workflows/build.yml

```diff
@@ -158,7 +165,12 @@ jobs:
         run: |
             mkdir build
             cd build
-            cmake -DCMAKE_INSTALL_PREFIX=appdir/usr/ -DCMAKE_BUILD_TYPE=Release -DPAJLADA_SETTINGS_USE_BOOST_FILESYSTEM=On ..
+            cmake \
+              -DCMAKE_INSTALL_PREFIX=appdir/usr/ \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DPAJLADA_SETTINGS_USE_BOOST_FILESYSTEM=On \
+              -DUSE_PRECOMPILED_HEADERS=${{ matrix.pch }} \
```

Comment:
These work with `true` and `false`, not only `On` or `Off`, right?

Reply:
Yes, verified by adding the message with PCH status

See https://github.com/Chatterino/chatterino2/runs/4054807765?check_suite_focus=true#step:17:63 and https://github.com/Chatterino/chatterino2/runs/4054807993?check_suite_focus=true#step:17:63